packages/browser/src/plugins/remote-loader/index.ts

```diff
@@ -86,6 +89,13 @@ export class ActionDestination implements DestinationPlugin {
           methodName,
           type: 'action',
         })
+
+        if (!(await this.ready())) {
+          throw new Error(
+            'Something prevented the destination from getting ready'
```

Comment:
Does this error get surfaced to the user somehow? ALso might be useful to include the name of the destination in this message.

Reply:
No it doesn't make to the surface, gets suppressed in one of the catch blocks in the call stack